fxa-auth-db-server/docs/API.md

```diff
@@ -35,6 +35,7 @@ All responses conform to the following:
 The following datatypes are used throughout this document:
 
 * epoch     : the date in ms since epoch. eg. 1424819640738 such as that returned from `Date.now()`
+* hex48     : a 64 bit value in hex encoding. ie. 12 chars long (6 bytes)
```

Comment:
Good catch, thanks. Although also, the length of signinCodes isn't actually mandated anywhere now that I'm storing the hash rather than the code itself. Maybe I shouldn't say anything about the length at all and just let the auth server say what it wants to about that.